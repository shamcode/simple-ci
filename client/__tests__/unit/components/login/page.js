import Page from '../../../../src/components/login/page';
import renderer from 'sham-ui-test-helpers';
import * as directives from 'sham-ui-directives';

it( 'renders correctly', () => {
    const meta = renderer( Page, {
        directives
    } );
    expect( meta.toJSON() ).toMatchSnapshot();
} );
