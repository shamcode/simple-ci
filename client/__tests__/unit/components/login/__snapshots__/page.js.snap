// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Object {
  "Options": Object {
    "dataSaving": false,
    "errors": Array [],
    "password": "",
  },
  "html": "
<!--0-->
<!--0-->
<div class=\\"login-form\\">
  <form>
    <div class=\\"group\\">
      <input name=\\"username\\" type=\\"text\\" required=\\"\\"><span class=\\"highlight\\"></span><span class=\\"bar\\"></span>
      <label>Username</label>
    </div>
    <div class=\\"group\\">
      <input name=\\"password\\" type=\\"password\\" required=\\"\\"><span class=\\"highlight\\"></span><span class=\\"bar\\"></span>
      <label>Password</label>
    </div>
    <div class=\\"group\\">
      <button type=\\"submit\\"><span>Login</span></button>
    </div>
  </form>
</div>
",
}
`;
